ThroughputUp_LinuxSharedEC_x64_openssl_Default_Loopback is Failing

### Describe the bug

With the recent change to use Linux SharedEC bits to run perf test, it seems to be consistently failing to run over loopback (a crash?):

https://dev.azure.com/ms/msquic/_build/results?buildId=318528&view=logs&j=4b0e614b-9a7e-5c6a-71f6-75937be47ea3&t=c5def7c4-164c-5bd8-800f-479e76b3b206

```PowerShell
Running Test ThroughputUp_LinuxSharedEC_x64_openssl_Default_Loopback
Test Run Took 00:01:56.2597028
Get-TestResult: /home/test/azpagent/_work/2/s/scripts/performance.ps1:476
Line |
 476 |  … edResults = Get-TestResult -Results $LocalResults -Matcher $Test.Resu …
     |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Error Processing Results:  Started! 

##[error]PowerShell exited with code '1'.
```

### Affected OS

- [ ] All
- [ ] Windows Server 2022
- [ ] Windows 11
- [ ] Windows Insider Preview (specify affected build below)
- [X] Ubuntu
- [ ] Debian
- [ ] Other (specify below)

### Additional OS information

_No response_

### MsQuic version

main

### Steps taken to reproduce bug

Run secnetperf loopback tests with sharedec Linux.

### Expected behavior

It should complete

### Actual outcome

It fails or crashes

### Additional details

_No response_

--- a/.azure/azure-pipelines.perf.yml
+++ b/.azure/azure-pipelines.perf.yml
@@ -451,7 +451,7 @@ stages:
         extraTestArgs: -ExtraArtifactDir _SharedEC
         ${{ if ne(parameters.testToRun, 'all') }}:
           testToRun: ${{ parameters.testToRun }}
-        testTypes: ${{ parameters.testTypes }}
+        testTypes: 'Remote'
         ${{ if eq(parameters.pgo_mode, false) }}:
           extraArgs: -SharedEC -Publish
         ${{ if eq(parameters.pgo_mode, true) }}:

